vss-processor-plugin/build.gradle.kts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+repositories {
+    gradlePluginPortal()
+    google()
+    mavenCentral()
+}
+
+plugins {
+    `kotlin-dsl`
+    `java-gradle-plugin`
+    publish
```

Comment:
There is a separate "publish plugin" for Gradle Plugins, try using this instead of the normal publish plugin, see here: 
https://github.com/JLLeitschuh/ktlint-gradle/blob/main/plugin/build.gradle.kts#L10
https://github.com/detekt/detekt/blob/main/detekt-gradle-plugin/build.gradle.kts#L15

Reply:
This seems to be intended for publishing to the public gradle plugin portal which is fine to ignore for now IMO since we all other libraries are also published to our own Maven Nexus.